app/components/chart/bgSourceToggle.js

```diff
@@ -1,62 +1,64 @@
-import React, { PureComponent } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import _ from 'lodash';
 import { components as vizComponents, utils as vizUtils } from '@tidepool/viz';
 
-import { BG_DATA_TYPES } from '../../core/constants';
-
 const { TwoOptionToggle } = vizComponents;
 const { statBgSourceLabels } = vizUtils.stat;
 
-class BgSourceToggle extends PureComponent {
```

Comment:
No functional changes.  Just converting to a functional component prior to match the new sample interval toggle